templates/app/webpack-default/template/src/utils.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * This file contains various utility functions.
+ */
+
+/**
+ * Creates a new tab for a tab group applying various defaults and merging
+ * it with the specified options.
+ *
+ * @param {object} options Options for the new tab
+ * @return {object} The created tab
+ */
+export function createTab(options) {
+	const defaults = {
+		activeTintColor: '#E82C2A',
+		activeTitleColor: '#E82C2A'
+	};
+	const mergedOptions = Object.assign({}, defaults);
+	if (runs('android')) {
+		Object.assign(mergedOptions, {
+			titleColor: '#f8bbc2',
+			activeTitleColor: '#fff'
+		});
+		delete options.icon;
+	}
+	Object.assign(mergedOptions, options);
+	return Ti.UI.createTab(mergedOptions);
+}
+
+/**
+ * Checks if the current device runs the specified platform.
+ *
+ * @param {string} platform Platform name to check
+ * @return {boolean} True if current device is running on the specified platform, false if not.
+ */
+export function runs(platform) {
```

Comment:
So [appcelerator/babel-plugin-transform-titanium](https://github.com/appcelerator/babel-plugin-transform-titanium#special-defines) actually has `OS_IOS` and `OS_ANDROID` "constants" that it will pre-populate to guard platform-specific code. (It also tries to sniff some common ways of detecting platform like the `Ti.Platform.osname` checks against constants, and injecting values for `Ti.Platform.osname/name`)

It may be more straight-forward and lead to reduced dead-code to use those directly in the app rather than extract a method like this.

Reply:
Ah crap, i totally forgot those additions to our transform plugin. I'll switch it out to use the constants!